scripts/scripts.js

```diff
@@ -0,0 +1,93 @@
+const app = {};
+
+// asynchronous fetch of local JSON data
+app.getData = async () => {
```

Comment:
Interesting method of using promises to fetch JSON data!
Curious as to why.

Reply:
Does it have to do with the fact that requirements prevent you from using a bundler?

Reply:
The first requirement was "Your résumé information is stored in JSON", which I took literally rather than resume information stored as a JavaScript object (which other people did, and would've been easier). Using fetch made sense to me given that constraint, but it likely isn't the most elegant solution.

Reply:
Oh nice! Yea, I couldn't figure out that part.
My vanilla JS is rusty.